src/components/layout/Layout.tsx

```diff
@@ -32,7 +32,7 @@ const useStyles = makeStyles((theme) =>
     },
     offset: {
       ...theme.mixins.toolbar,
-      marginBottom: theme.spacing(6),
+      marginBottom: theme.spacing(0),
```

Comment:
As we reduce the margin here we might want to introduce a margin to all other pages as the title is stitched to the app bar.



![image](https://user-images.githubusercontent.com/893608/109347910-83f3c180-787c-11eb-8512-be0d059a2cb8.png)

Reply:
We can place a bool option to `<Layout disableOffset />`

```tsx
{!disableOffset && <div className={classes.offset} />}
```

And we get rid of the margin specific for the page